README.md

```diff
@@ -722,6 +725,15 @@ Please note that the default for the [attribute](#attributes) is: `false`.
 
 You can change your default setting in your [configuration](#configuration) with the key `defaults.remove_image`.
 
+### Excluded heading
```

Comment:
Rewrite to "Excluded headings", since we are describing the feature in generically